src/nupic/types/Sdr.hpp

```diff
@@ -506,6 +506,23 @@ class SparseDistributedRepresentation : public Serializable
      */
     void load(std::istream &inStream) override;
 
+
+    // New serialization library **Cereal**.
+    // Do NOT call this externally!  Instead use methods save/load.
```

Comment:
good warning, but make the methods `private:` for that reason, and call-em from save/load.

Reply:
The Cereal methods cannot be in private, it will break Cereal.
When we got Cereal completely implemented, it will replace save/load.  Actually it will use the names save/load.  They are save_ar and load_ar now only because right now it conflicts with the existing routines.

Reply:
There is actually a way to make private save load methods with cereal, search the cereal doxygen docs for keywork: access.

Reply:
yes, I do realize that.  But I don't think it is necessary.